ui/jobs/bars.ts

```diff
@@ -729,7 +732,7 @@ export class Bars {
     if (!canShow || showAfterMs > 0) {
       this.o.leftBuffsList.removeElement('foodbuff');
       if (canShow)
-        return window.setTimeout(this._updateFoodBuff.bind(this), showAfterMs);
+        return window.setTimeout(this._updateFoodBuff.bind(this, o), showAfterMs);
```

Comment:
I said this to Maiko, but I think this is also incorrect in a different way.  It prevents the `undefined` bug, but the state in `o` is not necessarily correct any more.  What if the player is in combat or has changed zones into a different content type when this expires? `o` will still have the previous values that were originally passed in here.